site/content/en/docs/Guides/feature-stages.md

```diff
@@ -49,10 +49,10 @@ The current set of `alpha` and `beta` feature gates:
 | [GameServer Stable Network ID]({{% ref "/docs/Reference/gameserver.md#stable-network-id" %}})                         | `PodHostname`                  | Enabled  | `Beta`  | 1.32.0 |
 | [Reset Metric Export on Fleet / Autoscaler deletion]({{% relref "./metrics.md#dropping-metric-labels" %}})            | `ResetMetricsOnDelete`         | Enabled  | `Beta`  | 1.26.0 |
 | [Graceful Termination for GameServer SDK](https://github.com/googleforgames/agones/pull/2205)                         | `SDKGracefulTermination`       | Enabled  | `Beta`  | 1.18.0 |
+| [Split `agones-controller` ](https://github.com/googleforgames/agones/issues/2797)                                    | `SplitControllerAndExtensions` | Enabled  | `Beta`  | 1.32.0 |
```

Comment:
This change should be behind a release guard (even though the release is in like 2 days). 

https://agones.dev/site/docs/contribute/#within-a-page

Reply:
This should already be within the `{{% feature publishVersion="1.32.0" %}}` section. It was already created so I just made an edit to it.

Reply:
You are correct. Sorry I didn't look at the collapsed sections closely enough to see.